wordpress-mobile/WordPress-iOS
# Comments: support `blockEmailNotifications` from WPKit

Ref. #15642 
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/369

This PR updates the host app to support the new property `blockEmailNotifications` from WPKit. It also updates the data model to No. 115 and adds the `blockEmailNotifications` attribute to the `AccountSettings` entity.

There are no UI changes at this time.

### To test

1. `rake dependencies`
2. Build and run
3. Use a non-a8c account
4. On the web, [block email notifications](https://wordpress.com/me/notifications/subscriptions)
5. In the app, navigate to Reader > a post > comments and attempt to Follow the conversation by email.
6. A toast message should appear "Unable to follow conversation" (no changes to toast message)

PR submission checklist:
